source/localizable/_footer.html.erb

```diff
@@ -24,14 +24,14 @@
       </article>
     </div>
     <div class="footer-column footer-right">
-      <a href="https://thoughtbot.com?utm_source=middleman">
-        <span class="footer-thoughtbot-logo"></span>
+      <a href="https://formkeep.com?utm_source=middleman">
+        <span class="footer-formkeep-logo"></span>
       </a>
       <p class="footer-credit">
-        thoughtbot designed this site and is
+        FormKeep provides
 
-        <%= link_to "available for hire",
-          "https://thoughtbot.com/hire-us?utm_source=middleman" %>.
+        <%= link_to "form endpoints for designers and developers",
```

Comment:
How about "form endpoints for your Middleman sites" or "form backends for your Middleman sites"?